pkg/sql/logictest/testdata/logic_test/crdb_internal

```diff
@@ -619,14 +619,19 @@ start_pretty   end_pretty
 statement ok
 TRUNCATE TABLE foo
 
-# Ensure that there are no longer any splits left over on the original indexes.
-# TRUNCATE will have created equivalent splits points on the new indexes, so
-# this test just checks that there are no more on the old indexes.
+# Ensure splits are not unsplit right away with the transaction.
+# Instead, they are now unsplit in schema change gc jobs
```

Comment:
There are several job types, one of which is `GC` and another of which is `SCHEMA CHANGE GC`. Are you sure your unsplitting logic runs in the latter and not in the former? This isn't a rethorical question btw.